ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/AbstractPendingTransactionsSorter.java

```diff
@@ -256,8 +256,14 @@ protected void trackTransactionBySenderAndNonce(final TransactionInfo transactio
   protected void removeTransactionTrackedBySenderAndNonce(final Transaction transaction) {
     Optional.ofNullable(transactionsBySender.get(transaction.getSender()))
         .ifPresent(
-            transactionsForSender ->
-                transactionsForSender.removeTrackedTransaction(transaction.getNonce()));
+            transactionsForSender -> {
+              transactionsForSender.removeTrackedTransaction(transaction.getNonce());
+              if (transactionsForSender.transactionCount() == 0) {
+                LOG.trace(
+                    "Removing sender {} from transactionBySender since no more tracked transactions");
```

Comment:
There should be an arg for this trace call, or the `{}` should be removed.

Reply:
Added the argument